mypy/semanal_typeddict.py

```diff
@@ -276,9 +276,7 @@ def fail_typeddict_arg(self, message: str,
     def build_typeddict_typeinfo(self, name: str, items: List[str],
                                  types: List[Type],
                                  required_keys: Set[str]) -> TypeInfo:
-        fallback = (self.api.named_type_or_none('typing.Mapping',
-                                                [self.api.named_type('__builtins__.str'),
-                                                 self.api.named_type('__builtins__.object')])
+        fallback = (self.api.named_type_or_none('mypy_extensions._TypedDict', [])
```

Comment:
I would fail soon here if we can't find `mypy_extensions._TypedDict`. There is some logic than may cause a crash/fail that will be harder to debug.